static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx

```diff
@@ -257,24 +282,40 @@ const PresortedKeyValueList = ({data}: {data: KeyValueListData}) => (
   <KeyValueList shouldSort={false} data={data} />
 );
 
-const makeTransactionNameRow = (event: Event, orgSlug: string) => {
-  const to = transactionSummaryRouteWithQuery({
+const makeTransactionNameRow = (event: Event, orgSlug: string, projectSlug?: string) => {
```

Comment:
The reason why this is optional, is just to support the span evidence preview?

Reply:
Yep exactly, it's optional in the preview so we can't guarantee it is always present